### PR TITLE
[Feature] Added feature to allow users to select preferred styling lang

### LINKED ISSUE #205: ✨ [Feature request]: Allow selection of preferred styling language
**Is your feature request related to a problem? Please describe.**
Not a problem

**Describe the solution you'd like**
The command line input during play creation should allow the user to select their own choice of styling language (i.e.: CSS/SCSS) and accordingly a template file needs to be placed under the play directory.

**Describe alternatives you've considered**
NA

**Additional context**
NA

### PR DESCRIPTION
**Pull request by naiknareshh**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/atapas/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

This feature allows user to select preferred styling language while creating the play. an empty 'css' or 'scss' file will be automatically added in their play directory based on the selection. No dependencies were added for this change.

Fixes #205 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

1. Create new plays and alternatively select 'css' and 'scss' when asked for 'Styling to be used (css/scss):'
2. Validate if expected file is getting created in your play directory
3. Validate that expected file type is getting imported in your JSX file

# Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by naiknareshh** [comment]
This is Ready for Review :)

**Review by atapas — CHANGES_REQUESTED** [review]
Thanks @naiknareshh for picking this up.

Review comments given.

If you could post some screenshots of your testing would be great.

Additionally, you need to update the CREATE PLAY Read me file to include this option and change the screenshot there. Please check https://github.com/atapas/react-play/blob/main/CREATE-PLAY.md#%EF%B8%8F-steps-to-create-a-play

**Comment by naiknareshh** [comment]
## Testing Steps

### Step 1: Validate 'Style to be used' question is queued up as expected
image

### Step 2: Validate the files are getting created as expected and without any errors in the console
image

### Step 3: Validate proper import statement is added in the JSX file 
image

### Step 4: Add some styles to some existing element
image

image


### Step 5: Validate the styles are reflected properly on the play without any issues
image

File: plop-templates/component_ts.hbs [hunk 1]
```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import { getPlayById } from 'meta/play-meta-util';
 
 import PlayHeader from 'common/playlists/PlayHeader';
+import './{{pascalCase name}}.{{styling}}';
```
[the remark this answers is not in the document]
Comment (naiknareshh) [reply plop-templates/component_ts.hbs]: Agreed, fixed in the next commit.

File: plopfile.js [hunk 2]
```diff
@@ -33,6 +33,12 @@ module.exports = plop => {
         message: 'Language to be used (javascript/typescript):',
         choices: ['js', 'ts']
       },
+      {
+        type: 'list',
+        name: 'styling',
```
[the remark this answers is not in the document]
Comment (naiknareshh) [reply plopfile.js]: Agreed, fixed in the next commit.

File: plopfile.js [hunk 3]
```diff
@@ -71,6 +77,11 @@ module.exports = plop => {
         path: 'src/plays/{{generateFolderName name}}/{{pascalCase name}}.{{language}}x',
         templateFile: 'plop-templates/component_{{language}}.hbs',
       },
+      {
+        type: 'add',
+        path: 'src/plays/{{generateFolderName name}}/{{pascalCase name}}.{{styling}}',
```
[the remark this answers is not in the document]
Comment (naiknareshh) [reply plopfile.js]: fixed in the next commit.

**Comment by naiknareshh** [comment]
@atapas All comments have been incorporated, please review and let me know if there is anything.

**Review by atapas — APPROVED** [review]
@naiknareshh Looks good to me

**Comment by atapas** [comment]
@all-contributors please add @naiknareshh for Code

**Pull request merged**